vcs/src/main/java/org/openjdk/skara/vcs/Repository.java

```diff
@@ -183,6 +183,8 @@ default void updateSubmodule(Submodule s) throws IOException {
         updateSubmodule(s.path());
     }
 
+    boolean isEmptyCommit(Hash hash);
```

Comment:
We should add a javadoc comment describing the expected behavior for this method, specifically for merge commits.

Reply:
Sure. But I think it also works for simple empty commit(not merge commit)

Reply:
At first, I name this method as `isEmptyMergeCommit` and later I thought it also works for simple empty commit, so I changed the method name to `isEmptyCommit`

Reply:
NVM, I misunderstood your comment